src/Chainweb/WebPactExecutionService.hs

```diff
@@ -52,3 +59,19 @@ mkPactExecutionService q = PactExecutionService
         Right pdo -> return pdo
         Left e -> throwM e
   }
+
+-- TODO: to support mempool transaction reintroduction we need to hook into
+-- consensus instead of just killing every tx that ever made it into a valid
+-- block
+markAllConfirmed
+    :: MempoolBackend ChainwebTransaction
+    -> PayloadWithOutputs
+    -> IO ()
+markAllConfirmed mempool payload = mempoolMarkConfirmed mempool txHashes
+  where
+    txcfg = mempoolTxConfig mempool
+    decodeTx = codecDecode $ txCodec txcfg
```

Comment:
This is fine I guess, but this is perhaps why Pact service should be doing this, as it has the decoded transactions and could straight give you the hashes. Up to you

Reply:
I don't want this code to live in here past next week -- this was the smallest change set I could muster